### ISSUE TITLE
plotman plot  and  plotman interactive behaviour may be misleading

### ISSUE THREAD

**Issue by Carglglz** [ericaltendorf/plotman#579]
**Describe the bug**

Not exactly a bug but  the way that ``plotman plot`` and ``plotman interactive`` works may lead to confusion.
**To Reproduce**

Steps to reproduce the behavior, e.g.:
1. Set up config in ``plotman.yaml`` and run ``plotman plot``
2. ``plotman plot`` will load ``plotman.yaml`` configuration and start spawning new plots processes
3. Run ``plotman interactive``, change ``plotman.yaml``
4. ``plotman interactive`` will load new configuration and spawn plots according to that.
5. Quit ``plotman interactive``
6. When the plots scheduled by ``plotman interactive`` are done, new plots are scheduled by ``plotman plot`` with the old configuration 

**Expected behavior**

Explain this behaviour in the README to avoid confusion.

**Possible Solution**

Add and option like ``plotman plot -u`` to update the configuration every time it spawns a new plot.

I could submit a PR to fix this if necessary. 👍

**Comment by altendky** [comment]
I don't plan on enabling auto-config reloading.  I do plan on moving towards a more service-and-monitor style system.  It's not too far from what's there now, but there are some ragged edges to work on to make it tidy.  As to `interactive` or `plot` taking priority, it all depends what configuration each is running with and what the timing is as to which one ends up 'controlling' the plot launching.  This is a general issue that we will work towards solving.

**Comment by Carglglz** [comment]
> As to interactive or plot taking priority, it all depends what configuration each is running with and what the timing is as to which one ends up 'controlling' the plot launching. This is a general issue that we will work towards solving.

Yes I see that clearly now after looking at ``plot`` code. I wrongly assumed that ``interactive`` was updating ``plot`` configuration too.

**Comment by thomascooper** [comment]
I have a similar issue. I loaded interactive with "tmpdir_max_jobs: 2" . I noticed this after hitting 4 plots and seeing "no eligible tempdirs (4868s/1800s)". So I quit interactive mode, updated the config to 4 jobs and restarted interactive. There was no change, it still says "no eligible tempdirs". Do I need to wait for the current jobs to complete to have the status reset?

**Comment by thomascooper** [comment]
Actually I am going to answer my own question, I did not read the config well enough. I am not spawing new jobs because phase major and minor are 2:1. and only 1 job is allowed until this level is passed.
